packages/platform/lang/en.json

```diff
@@ -10,10 +10,10 @@
     "UnknownMethod": "Unknown method: {method}",
     "InternalServerError": "Internal server error",
     "MaintenanceWarning": "Maintenance Scheduled in {time, plural, =1 {less than a minute} other {# minutes}}",
-    "AccountNotFound": "Account not found",
+    "AccountNotFound": "Invalid credentials",
     "AccountNotConfirmed": "Account not confirmed",
     "WorkspaceNotFound": "Workspace not found",
-    "InvalidPassword": "Invalid password",
+    "InvalidPassword": "Invalid credentials",
```

Comment:
I feel it would be better if we get rid of this string all together and use AccountNotFound for both cases